cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,66 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
-
-  it('should have the ability to assert automatically resolved alerts', () => {
```

Comment:
It is a template for your test. Please don't remove it, but complete the test. Please check everywhere